File: hbe-tests/projects/TEST-11-appbundle/app/src/main/java/com/hbe/testaab/MainActivity.java
package com.hbe.testaab;

import android.app.Activity;
import android.os.Bundle;
import android.widget.TextView;

public class MainActivity extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        TextView text = findViewById(R.id.text);
        if (text != null) {
            text.setText(getString(R.string.app_name));
        }
    }
}

File: hbe-tests/projects/TEST-13-android36/app/src/main/kotlin/dev/hbe/test13/MainActivity.kt
package dev.hbe.test13

import android.app.Activity
import android.os.Bundle
import android.widget.TextView

class MainActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val tv = TextView(this)
        tv.text = "TEST-13 Android 36"
        setContentView(tv)
    }
}
